sim/dex.ts

```diff
@@ -102,6 +102,12 @@ interface DexTableData {
 	Statuses: DexTable<EffectData>;
 	TypeChart: DexTable<TypeData>;
 }
+interface TextTableData {
+	Abilities: AnyObject;
```

Comment:
I think you can give these a better type than AnyObject?

Reply:
I could but laziness.